src/Tezos/TWTezosSigner.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include <string>
+
+#include "proto/Tezos.pb.h"
+#include "Tezos/TWTezosForger.h"
+#include "HexCoding.h"
+#include <TrustWalletCore/TWHash.h>
+#include <TrustWalletCore/TWString.h>
+
+std::string signOperation(TW::Tezos::Proto::OperationList operationList) {
+  auto forgedBytesHex = forgeBranch(operationList.branch());
```

Comment:
I don't love auto here because it's not obvious from the function name or variable name, but that's just my opinion